apps/web/src/components/Menu/index.tsx

```diff
@@ -28,6 +30,8 @@ const Menu = (props) => {
   const cakePrice = useCakePrice()
   const { currentLanguage, setLanguage, t } = useTranslation()
   const { pathname } = useRouter()
+  const { featureEnabled } = useABTestingManager(FEATURE_FLAGS.WebNotifications)
```

Comment:
what about rename to `useExperimentalFeatureEnabled`? And as for `FEATURE_FLAGS` -> `EXPERIMENTAL_FEATURES`.

```typescript
const webNotificationEnabled = useExperimentalFeatureEnabled(EXPERIMENTAL_FEATURES.WebNotification)
```

Reason for this suggestion: when we say `useSomething`, we're expecting `something` to be the thing that we can from the hook. What we care about is whether an experimental feature is enabled or not, not an ab testing manager